HW_1/treugolnik.py

```diff
@@ -0,0 +1,12 @@
+a = float(input("Введите длину первой стороны треугольника: "))
+b = float(input("Введите длину второй стороны треугольника: "))
+c = float(input("Введите длину третьей стороны треугольника: "))
+
+if a >= b + c or b >= a + c or c >= a + b:
+    print("Треугольник с такими сторонами не существует!")
+elif a == b == c:
+    print("Треугольник является равносторонним")
+elif a == b or b == c or a == c:
+    print("Треугольник является равнобедренным")
+else:
+    print("Треугольник является разносторонним")
```

Comment:
Можно чуть укоротить условие, поменяв его на другое
```suggestion
elif a != b != c:
    print("Треугольник является разносторонним")
else:
    print("Треугольник является равнобедренным")
```